web/modules/custom/joinup_licence/README.md

```diff
@@ -0,0 +1,13 @@
+Joinup Licence
```

Comment:
This shows which graph to use to import the licences in Joinup. If we don't have this here then we are forcing people to go dig in the config files for this information.

We cannot put this in the SPDX module since the graph `http://joinup.eu/spdx_licence/published` is specific to Joinup and doesn't make sense in any other project that uses the SPDX module.